x-pack/plugins/profiling/public/components/no_data_page.tsx

```diff
@@ -265,43 +268,71 @@ docker.elastic.co/observability/profiling-agent:${hostAgentVersion} /root/pf-hos
       steps: [
         {
           title: i18n.translate('xpack.profiling.tabs.symbols.step1', {
-            defaultMessage: 'Download the symbtool binary from the URL below:',
+            defaultMessage: 'Download and extract symbtool',
           }),
           content: (
-            <EuiLink target="_blank" href={`https://ela.st/symbtool-linux-amd64`}>
-              https://ela.st/symbtool-linux-amd64
-            </EuiLink>
+            <EuiCodeBlock paddingSize="s" isCopyable>
+              {`# x86_64\nwget -O symbtool-amd64.tgz "https://ela.st/symbtool-linux-amd64" && tar xzf symbtool-amd64.tgz && cd symbtool-*\n\n # ARM64\nwget -O symbtool-arm64.tgz "https://ela.st/symbtool-linux-arm64" && tar xzf symbtool-arm64.tgz && cd symbtool-*`}
+            </EuiCodeBlock>
```

Comment:
```suggestion
            <EuiText>
              <b>For x86_64:</b>
              <EuiCodeBlock paddingSize="s" isCopyable>
                {`wget -O symbtool-amd64.tgz "https://ela.st/symbtool-linux-amd64" && tar xzf symbtool-amd64.tgz && cd symbtool-*-linux-x86_64`}
              </EuiCodeBlock>
              <EuiSpacer size="m" />
              <b>For ARM64:</b>
              <EuiCodeBlock paddingSize="s" isCopyable>
                {`wget -O symbtool-arm64.tgz "https://ela.st/symbtool-linux-arm64" && tar xzf symbtool-arm64.tgz && cd symbtool-*-linux-arm64`}
              </EuiCodeBlock>
            </EuiText>
```

I separated out the commands that need to be copied for both architectures. This should help the user only copy the command needed for their given architecture.

I also added the operating system and architecture to both commands. I saw the error `cd: too many arguments` because there are two paths in the local directory starting with `symbtool-` if either command is executed.

Reply:
Good suggestion, applied that! I additionally added an `<EuiSpacer>` between the `<b>` text and the code blocks since it looked a bit too close without that.